mlflow/models/evaluation/base.py

```diff
@@ -456,11 +485,121 @@ def _start_run_or_reuse_active_run(run_id):
+            print(f'Hint: Multiple registered evaluators are found {evaluator_name_list} and '
+                  'they will all be used in evaluation. If you want to evaluate with one '
+                  'evaluator, specify the `evaluator` argument and (optional) specify the '
+                  '`evaluator_config` argument.')
+        if evaluator_config is not None:
+            conf_dict_value_error = ValueError(
+                "If `evaluators` argument is None, all registered evaluators will be used, "
+                "if only default evaluator available, the `evaluator_config` argument can be "
+                "config dict for default evaluator, otherwise the `evaluator_config` argument "
+                "must be a dict contains mapping from evaluator name to individual "
+                "evaluator config dict."
+            )
+            if evaluator_name_list == ['default']:
+                if not isinstance(evaluator_config, dict):
+                    raise conf_dict_value_error
+                elif 'default' not in evaluator_config:
+                    evaluator_name_to_conf_map = {'default': evaluator_config}
+                else:
+                    evaluator_name_to_conf_map = evaluator_config
+            else:
+                if not check_nesting_config_dict(evaluator_name_list, evaluator_config):
+                    raise conf_dict_value_error
+                evaluator_name_to_conf_map = evaluator_config
+        else:
+            evaluator_name_to_conf_map = {}
+    elif isinstance(evaluators, str):
+        if not (evaluator_config is None or isinstance(evaluator_config, dict)):
+            raise ValueError(
+                "If `evaluators` argument is the name of an evaluator, evaluator_config must be "
+                "None or a dict containing config items for the evaluator."
+            )
+        evaluator_name_list = [evaluators]
+        evaluator_name_to_conf_map = {evaluators: evaluator_config}
+    elif isinstance(evaluators, list):
+        if evaluator_config is not None:
+            if not check_nesting_config_dict(evaluators, evaluator_config):
+                raise ValueError(
+                    "If `evaluators` argument is a evaluator name list, evaluator_config "
+                    "must be a dict contains mapping from evaluator name to individual "
+                    "evaluator config dict."
+                )
+        evaluator_name_list = list(set(evaluators))
+        evaluator_name_to_conf_map = evaluator_config or {}
+    else:
+        raise ValueError(
+            '`evaluators` argument must be None, a evaluator name string, or a list of '
+            'evalautor names.'
+        )
+
+    return evaluator_name_list, evaluator_name_to_conf_map
+
+
+def _evaluate(
+        model, model_type, dataset, actual_run_id, evaluator_name_list, evaluator_name_to_conf_map
+):
+    """
+    This method is the patch point for databricks instrumentation.
```

Comment:
Can we remove this part of the docstring so that OSS developers don't have to read / reason about Databricks? Our internal testing will be enough to verify that this method remains unaffected.